todoapp/flask_config.py

```diff
@@ -7,3 +7,12 @@ class Config:
     DB_URL = os.environ.get('DB_URL')
     if not DB_URL:
         raise ValueError(f"No DB_URL set for Flask application or corrupt: [{DB_URL}] ?")
+
+    LOG_LEVEL = os.environ.get('LOG_LEVEL')
+    if not LOG_LEVEL:
+        LOG_LEVEL="ERROR"
```

Comment:
Or this?
```suggestion
    LOG_LEVEL = os.environ.get('LOG_LEVEL', 'ERROR')
```

Or to exactly match the behaviour, even for LOG_LEVEL set to empty string, you could do `LOG_LEVEL = os.environ.get('LOG_LEVEL') or 'ERROR'`